### ISSUE TITLE
Display of ROOTFile fails for certain file

### ISSUE THREAD

**Issue by Moelf** [tamasgal/UnROOT.jl#36]
```
ROOTFile with 2 entries and 25 streamers.
./test/samples/km3net_offline.root
ERROR: unlock count must match lock count
Stacktrace:
  [1] error(s::String)
    @ Base ./error.jl:33
  [2] unlock(rl::ReentrantLock)

...
caused by: UndefVarError: Head not defined
Stacktrace:
  [1] macro expansion
    @ ~/.julia/dev/UnROOT/src/root.jl:111 [inlined]
  [2] (::UnROOT.var"##getter#258#116"{ROOTFile, String})()
    @ UnROOT ~/.julia/packages/Memoization/f2NeI/src/Memoization.jl:96
  [3] get!(default::UnROOT.var"##getter#258#116"{ROOTFile, String}, lru::LRUCache.LRU{Any, Any}, key::Tuple{Tuple{ROOTFile, String}, NamedTuple{(), Tuple{}}})
```

**Comment by 8me** [comment]
For the record: the locking issue occurring there should have been solved by [df2b889c5a7d8f8efda6b0522cc05f2c99bf8c9a](https://github.com/tamasgal/UnROOT.jl/commit/df2b889c5a7d8f8efda6b0522cc05f2c99bf8c9a)

**Comment by tamasgal** [comment]
That's good news! We should take the opportunity to loop over all sample files in the test suite and make sure that printing is working, or at least does not error out 🤪
